cumulusci/cli/project.py

```diff
@@ -218,6 +210,19 @@ def project_init(runtime):
             click.style("Minimum code coverage percentage", bold=True), default=75
         )
 
+    init_from_context(context)
+
+
+def init_from_context(context: Dict[str, object]):
```

Comment:
I added some minimal type-hinting but we can be more strict if needed:

- We could use a [`TypedDict`](https://docs.python.org/3/library/typing.html#typing.TypedDict) to explicitly list out what key/value pairs are expected in `context`
- We could use actual keyword arguments instead of a dict

Reply:
@jerivas we've been using pydantic models more and more, so I would consider that option as well. We're still supporting Python 3.6 so TypedDict isn't an option yet. In the end I don't have strong feelings about it though.